go.mod

```diff
@@ -2,6 +2,10 @@ module sigs.k8s.io/azurefile-csi-driver
 
 go 1.23.1
 
+godebug winreadlinkvolume=0
```

Comment:
note that we _may_ just be setting the `winsymlink` option in 1.32 in Kubernetes, see discussion in https://github.com/kubernetes/kubernetes/pull/129103